README.md

```diff
@@ -56,6 +56,19 @@ Updated Endpoint: `Create CIS Deductions for Subcontractor`
 
 ### individual-calculations-api
 
+* Some error response status codes corrected from 403 to 400, see [sandbox changelog](https://github.com/hmrc/income-tax-mtd-changelog/wiki#10th-january-2023).
+* Version 2.0 has been deprecated see [sandbox changelog](https://github.com/hmrc/income-tax-mtd-changelog/wiki#9th-january-2023)
+* New Gov-Test-Scenario header values added for `Trigger a Self Assessment Tax Calculation` endpoint [sandbox changelog](https://github.com/hmrc/income-tax-mtd-changelog/wiki#1-december-2022)
+* New fields have been added for `Retrieve a Self Assessment Tax Calculation` endpoint [sandbox changelog](https://github.com/hmrc/income-tax-mtd-changelog/wiki#5th-january-2023)
+* New error code has been added for `Submit a Self Assessment Final Declaration` endpoint [sandbox changelog](https://github.com/hmrc/income-tax-mtd-changelog#8-march-2023)
+
+Updated Endpoint: `List Self Assessment Tax Calculations`
+
+* Updated description for the endpoint
+* Updated description for `calculationTimestamp` field
+* Removed `biss` from `	calculationType` field
```

Comment:
I'm not sure what that means. Was it previously "bissCalculationType"?  If so, maybe: "Renamed the bissCalculationType field to calculationType"

Reply:
(actually there's also currently a space between the tick-mark and calculationType)

Reply:
It's from [here](https://github.com/hmrc/individual-calculations-api/pull/409/files), it used to have `biss` as one of the calculation types but now it has been removed. 
<img width="1251" alt="Screenshot 2023-03-28 at 13 02 33" src="https://user-images.githubusercontent.com/108878081/228231723-61c1f019-53c2-4412-bd81-aa305c81a20b.png">